src/Input/Silk.NET.Input.Common/Interfaces/IKeyboard.cs

```diff
@@ -39,5 +39,17 @@ public interface IKeyboard : IInputDevice
         /// Called when a character is received.
         /// </summary>
         event Action<IKeyboard, char>? KeyChar;
+
+        /// <summary>
+        /// Begins taking keyboard input. Required on mobile, where keyboard input is usually delivered via an on-screen
+        /// keyboard.
+        /// </summary>
+        void BeginInput();
+
+        /// <summary>
+        /// Ends taking keyboard input. Required on mobile, where this method closes the on-screen keyboard.
+        /// </summary>
+        /// <seealso cref="BeginInput"/>
+        void EndInput();
```

Comment:
Should probably document what happens when this is done on Desktop.

Reply:
```suggestion

        /// <summary>
        /// Begins taking keyboard input. Required on mobile, where keyboard input is usually delivered via an on-screen
        /// keyboard.
        /// Not required on desktop (it does nothing), but recommended to ensure easy porting to other platforms.
        /// </summary>
        void BeginInput();

        /// <summary>
        /// Ends taking keyboard input. Required on mobile, where this method closes the on-screen keyboard.
        /// Not required on desktop (it does nothing), but recommended to ensure easy porting to other platforms.
        /// </summary>
        /// <seealso cref="BeginInput"/>
        void EndInput();
```